pyaerocom/io/readungriddedbase.py

```diff
@@ -15,6 +15,7 @@
 
 logger = logging.getLogger(__name__)
 
+
```

Comment:
I see that a few files got extra empty lines.
Do this changes come from black?

Reply:
yes, these mini changes came from a newer version of black. Without the black update CI linting was failing. I might have just updated to a too new version...